### DISCUSSION TITLE
How to specify the link of left and right on GitHub

### DISCUSSION THREAD

**Discussion by JeffreyYou** [badges/shields#5593] [Q&A] [answered]
:question: How to specify the link of badge?

I'm trying to creat a badge which left side click go to Github and right side click go to Youtube.
Currently I have:
![](https://img.shields.io/badge/Github-Youtube-red?link=http://Github.com&link=http://youtube.com&logo=Github)
`https://img.shields.io/badge/Github-Youtube-red?link=http://Github.com&link=http://youtube.com&logo=Github`

But there is no hyperlink when I click it, how do I achieve this?

Thank you!

**Comment by paulmelnikow** [comment, marked as answer]
Hi! The links are there. If you open the SVG in its own window you can see that you can click through to YouTube and GitHub.

For the links to work in a web page, you need to embed the image using `<object>` rather than `<img>`.

On GitHub, you simply can't do it. The link needs to be in the Markdown. Links embedded in SVG simply will not work.

I think they're [stripped out by camo](https://docs.github.com/en/github/authenticating-to-github/about-anonymized-image-urls). This is a long-standing security choice made by GitHub. There have been discussions about this over the years but I can't put my hands on them at the moment. Though an interesting bit of history is the PR that made _logos_ work in GitHub, atmos/camo#74, which came out of the work done in #337.

**Reply by JeffreyYou** [reply]
It works prefectly outside the Github, now I understand why this issue happened.
Thank you for your reply.
